README.md

```diff
@@ -1,81 +1,127 @@
-![Terraform Provider Iterative](https://static.iterative.ai/img/cml/banner-terraform.png)
+![TPI](https://static.iterative.ai/img/cml/banner-tpi.svg)
 
-# Iterative Provider [![](https://img.shields.io/badge/-documentation-5c4ee5?logo=terraform)](https://registry.terraform.io/providers/iterative/iterative/latest/docs)
+# Terraform Provider Iterative (TPI)
```

Comment:
I know it's beyond scope but is a preposition missing from the tool's expanded name? I.g. Terraform Provider **for** Iterative (Tools)? "Iterative's Terraform Provider" would also be more readable IMO but at this point changing the TLA is prob. out of the question.

Reply:
Also on the title. The confusion is reinforced by the [official docs](https://registry.terraform.io/providers/iterative/iterative/latest/docs) site calling it just "iterative provider" (top of nav).